tests/configs/cycle/input/config.yaml

```diff
@@ -6,6 +6,8 @@ lab:
     small:
       cpu: 1.0
       memory: 3Gi
+fileserver:
+  image: docker.io/lsstsqre/worblehat
```

Comment:
If we make the file server optional, the specialty configs for testing other things won't need (and shouldn't have) file server configurations.

Reply:
will fix

Reply:
Done